### ISSUE TITLE
Web suite reports Device is in unexpected mode when using old FW in model One

### ISSUE THREAD

**Issue by bosomt** [trezor/trezor-suite#3181]
**Describe the bug**
+ Web suite /it doesn't happen via app/ reports `Device is in unexpected mode` when you connect device with old FW.
+ You have 100+ errors in browser console in ine minute
+ wallet.trezor.io works fine and updates device to FW 1.6.3 , so if we want to web suite to replace old wallet we need to properly handle device that wasn't yet upgraded

**Info:**
 - Suite Version: web, commit be4e3a36f7dbe15b2b203f66de11edc50a6665b8
 - Browser: Chromium , version 87
 - OS: Nixos
 - Firmware Version: 1.5.2 model One, revision 0xe4cc08775fc9c204f295442f930326eb7877f2d4,
 - Bootloader version 1.3.2
 - Bridge Version: 2.0.28

**Steps to reproduce:**
1. start web version of Suite
2. Connect device with FW older than version 1.6.3 
3. observe

**Expected result:**
Suite can handle older firmware versions.

**Actual result:**
Suite is not able to handle device that doesn't send features status
 
**Screenshots:**

![image](https://user-images.githubusercontent.com/31506317/102184404-b3b34000-3eaf-11eb-8e48-6f5ca6e3087f.png)
![image](https://user-images.githubusercontent.com/31506317/102185486-6a63f000-3eb1-11eb-98a9-393cf3febb4e.png)

**Additional info**

 trezorctl get_features
WebUSB transport is disabled: libusb-1.0.so: cannot open shared object file: No such file or directory
/home/tbosansk/Downloads/trezor-firmware/python/src/trezorlib/client.py:241: UserWarning: Firmware is out of date
  self.check_firmware_version(warn_only=True)
Features (124 bytes) {
    bootloader_hash: 32 bytes 0x7651b7caba5aae0cc1c65c8304f760396f77606cd3990c991598f0e22a81e007,
    device_id: '276DE4064BEBAA211510306C',
    flags: 0,
    imported: False,
    initialized: False,
    major_version: 1,
    minor_version: 5,
    needs_backup: False,
    passphrase_protection: False,
    patch_version: 2,
    pin_protection: False,
    revision: 20 bytes 0xe4cc08775fc9c204f295442f930326eb7877f2d4,
    unlocked: False,
    vendor: 'bitcointrezor.com',

**Comment by mroz22** [comment]
I'll look at it, but I'd say there must be some other reason of this. Not only that firmware is old. I definitely tested upgrading from old firmwares using freshly packed devices. Or maybe 1.5.2 is even older?

**Comment by bosomt** [comment]
model One id distributed with newer firmware than 1.5.2

**Comment by tsusanka** [comment]
@bosomt could you retest with Bridge 2.0.27? Because that's what's bundled inside the app and 28 contained a bug AFAIK.

**Comment by bosomt** [comment]
@tsusanka same results in Windows with default 2.0.27 installed at https://suite.trezor.io/web/

**Comment by tsusanka** [comment]
Just to clear up confusion: we do not ship the devices with any firmware. If the device is brand new we install 1.6.3 and then latest which is what @mroz22 meant and that should work without any problems.

That being said we should also deal with even older firmwares (old users) as @bosomt is rightfully bringing up. I will discuss in Product which firmwares we want to support (at least upgrade to latest).

**Comment by mroz22** [comment]
but if this works in desktop app then this must be either something with bridge or some kind of weird race condition. There is no difference between desktop and web other than bridge that could cause this I'd say :thinking:

**Comment by tsusanka** [comment]
@bosomt could you retest this? There were some changes so I am wondering if this error still prevails.

**Comment by bosomt** [comment]
@tsusanka I'm still not able to upgrade device with 1.6.0 firmware , i will record and attach video later

**Comment by bosomt** [comment]
Trying to update from 1.6.0 will get you to infinite loop.

Edit: attached log from latest app  [trezor-suite-log-1618935704620.txt](https://github.com/trezor/trezor-suite/files/6345125/trezor-suite-log-1618935704620.txt)

```

trezorctl get_features
Features (30 bytes) {
    bootloader_mode: True,
    firmware_present: True,
    major_version: 1,
    minor_version: 3,
    patch_version: 1,
    vendor: 'bitcointrezor.com',


tbosansky@UX433FA:~/Downloads$ trezorctl get_features
/usr/lib/python3/dist-packages/trezorlib/client.py:238: UserWarning: Firmware is out of date
  self.check_firmware_version(warn_only=True)
Features (132 bytes) {
    bootloader_hash: 32 bytes 0x1585215bc6e55a3407a8b3eee279034e95b9c4340033e1b6ae160ce661196715,
    device_id: '2BEE2A5CC2839B3DDF26F9E1',
    flags: 1,
    imported: False,
    initialized: True,
    label: 'kumoto',
    major_version: 1,
    minor_version: 6,
    needs_backup: False,
    passphrase_protection: False,
    patch_version: 0,
    pin_protection: True,
    revision: 20 bytes 0x723cf295a72ce07b96047901bb8c2e461a2488f8,
    unlocked: False,
    vendor: 'bitcointrezor.com',


```

https://user-images.githubusercontent.com/31506317/115372364-18b6d380-a1cb-11eb-969e-d9888aa95b01.mp4

**Comment by tsusanka** [comment]
Thanks! There definitely needs to be a way to update from 1.6.0. @slowbackspace something to check after the new onboarding is done?

**Comment by slowbackspace** [comment]
@tsusanka there wasn't much of the logic that changed in newer onboarding. If it doesn't work in old one, it won't work in new one, which was mostly presented as design refresh :)

We've briefly tried to troubleshoot this with boso, but no luck. Someone needs physical access to such device to be able to debug this properly 😁

**Comment by jerechinsky** [comment]
@trezor/qa Have you perhaps experienced this recently? Could you please try to replicate?

**Comment by STew790** [comment]
I tried it with a freshly unpacked device and installed FW 1.5.2. via trezorctl but still got the same error, as can be seen in the screenshot. Let me know if you need anything else.

**INFO**

- Suite version: web 21.7.1 (24cc565a3badd85ab48b30348022471dfd85f90e)
 - Browser: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.164 Safari/537.36
 - OS: Win32
 - Screen: 1440x900
 - Device: model 1 1.5.2
 - Bridge 2.0.27 
 
 
![image](https://user-images.githubusercontent.com/48477147/126983534-a01df060-ddd0-492e-928b-7dccdf72385e.png)

**Comment by tsusanka** [comment]
@STew790 same thing if you enter the device into the bootloader?

**Comment by STew790** [comment]
@tsusanka  Yes, same thing except there are no errors in console and Suite says that the device is in BL mode.
![image](https://user-images.githubusercontent.com/48477147/127004038-3ba97d0e-017a-459a-b81b-a155710e22a5.png)

**Comment by STew790** [comment]
Ok, I really don't know what happened, because I didn't change anything with my setup, but I tried again to connect the device in "normal" mode and it works all of a sudden and the Suite shows me an update from 1.5.2 to 1.10.2.

![image](https://user-images.githubusercontent.com/48477147/127005368-139fbc42-e34e-41f6-bb21-45f00c30a660.png)

**Comment by tsusanka** [comment]
> @tsusanka Yes, same thing except there are no errors in console and Suite says that the device is in BL mode.

Ok, never mind. Just a note: we do not need to fix Suite to allow working with 1.5.2 but we need to fix it to allow _updating_ to latest.

**Comment by tsusanka** [comment]
@STew790 didn't you use webusb the first time and Bridge the second time maybe?

**Comment by STew790** [comment]
@tsusanka Nope, I downloaded ver. 2.0.27 of Bridge prior to this actually, because Suite actually wants you to download and install it.

**Comment by tsusanka** [comment]
@STew790 retest please.

**Comment by STew790** [comment]
**QA OK**

I tested this on Linux as well as on Windows 10, both app and web version, and I got immediately prompted to install new FW. This was most probably solved by implementing the new onboarding. Closing.

![image](https://user-images.githubusercontent.com/48477147/137903825-91f4ded0-1c1e-49fc-8abe-2e6d706c23bc.png)


**Info:**
 - Suite version: web 21.10.2 (3cb57ead18474b380841b479a542c0b910b3d936)
 - Browser: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/94.0.4606.81 Safari/537.36
 - OS: Linux x86_64
 - Screen: 1920x1080
 - Device: model 1 1.5.2
